src/com/facebook/buck/apple/AppleLibraryDescription.java

```diff
@@ -474,6 +475,7 @@ public static boolean isSharedLibraryTarget(BuildTarget target) {
   public static class Arg extends AppleNativeTargetDescriptionArg {
     public Optional<SourcePath> infoPlist;
     public ImmutableMap<String, String> infoPlistSubstitutions = ImmutableMap.of();
+    public ImmutableList<String> swiftCompilerFlags = ImmutableList.of();
```

Comment:
I think this should be in `AppleNativeTargetDescriptionArg` as binary and test rule might need it too.